apps/web/components/booking/BookingListItem.tsx

```diff
@@ -185,7 +186,7 @@ function BookingListItem(booking: BookingItemProps) {
 
   const location = booking.location || "";
 
-  const onClick = () => {
+  const onClickTableData = (ev: React.MouseEvent<HTMLDivElement>) => {
```

Comment:
@JeroenReumkens Renamed to better understanding and don't collide with onClick.

Reply:
@alannnc  Cool! Main reason I added it in here instead of the default behavior of the Popover is because the click behavior is actually something the BookingListItem tries to control, and not something the Popover should know about. But I sticked with your suggestion and added it to the tooltip button as well. Abstracted it to a function to so I could add a comment as to why this is needed 👍